utils/frame/try-runtime/cli/src/parse.rs

```diff
@@ -42,3 +44,12 @@ pub(crate) fn url(s: &str) -> Result<String, &'static str> {
 		Err("not a valid WS(S) url: must start with 'ws://' or 'wss://'")
 	}
 }
+
+pub(crate) fn state_version(s: &str) -> Result<StateVersion, &'static str> {
+	match s.parse::<u32>() {
```

Comment:
You could reuse the `TryFrom<u8> for StateVersion` from `sp-storage` here.

Reply:
done